src/styles/functions.scss

```diff
@@ -0,0 +1,4 @@
+@function size($width, $gap) {
+  @return ($width * 8) + ($gap * 7);
```

Comment:
Don't got what this numbers 8 and 7 means and why we use it, if it's something important it's better to make variables with understandable names.